lib/python/vdm/server/HTTPListener.py

```diff
@@ -1806,6 +1841,9 @@ def main(runner, amodule, config_dir, data_dir, server):
     APP.add_url_rule('/api/1.0/databases/<int:database_id>/servers/recover',
                      view_func=RECOVER_DATABASE_SERVER_VIEW, methods=['PUT'])
 
+    APP.add_url_rule('/api/1.0/databases/<int:database_id>/servers/rejoin',
```

Comment:
This should have server id in the URL similar to START_DATABASE_SERVER_VIEW url. serverId should NOT be a request parameter.